src/gui/transferlistsortmodel.h

```diff
@@ -64,7 +66,9 @@ class TransferListSortModel final : public QSortFilterProxyModel
 
     TorrentFilter m_filter;
     mutable CachedSettingValue<int> m_subSortColumn;
+    mutable CachedSettingValue<int> m_subSortColumnOrder;
```

Comment:
How about rename it to `m_subSortOrder` and `m_lastSortOrder` respectively?
And storage key would become `TransferList/SubSortOrder`.

Reply:
renamed